invenio_communities/templates/invenio_communities/mycommunities.html

```diff
@@ -10,10 +10,13 @@
 <div class="well">
 {% if current_user.is_authenticated %}
   {% if mycommunities %}
+  {% if not can_user_create_community(current_user) %}
```

Comment:
I would extract this as a variable at the beginning of the template:

```jinja
{% set can_create_community = can_user_create_community(current_user) %}
```

and then use appropriately below